Core/LCG/GGUSTicketsClient.py

```diff
@@ -1,41 +1,52 @@
-# $HeadURL$
 """ GGUSTicketsClient class is a client for the GGUS Tickets DB.
 """
 
 import urllib2
 
-from suds        import WebFault
+from suds import WebFault
 from suds.client import Client
 
-from DIRAC import S_OK, S_ERROR
+from DIRAC import gConfig, S_OK, S_ERROR
 
 __RCSID__ = "$Id$"
 
-class GGUSTicketsClient:
 
-  # create the URL to get tickets relative to the site ( opened only ! ): 
+def getGGUSURL(vo=None, siteName=None):
+  """ create the URL to get tickets relative to the site ( opened only ! ):
+  """
+  if vo is None:
+    gConfig.getOption('DIRAC/VirtualOrganization')
```

Comment:
Did you forget to assign the result of the getOption() call ?

Reply:
Thanks, fixed.